radio/src/translations/se.h

```diff
@@ -1142,15 +1170,21 @@
 
 #define TR_INVERTED_SERIAL              INDENT "Invert"
 #define TR_IGNORE_INSTANCE              TR(INDENT "Ej inst.", INDENT "Ignorera instansfel")
-#define TR_DISCOVER_SENSORS             "Sök nya sensorer"
+#define TR_DISCOVER_SENSORS             "Sök nya"
 #define TR_STOP_DISCOVER_SENSORS        "Avbryt sökning"
 #define TR_DELETE_ALL_SENSORS           "Ta bort alla sensorer"
 #define TR_CONFIRMDELETE                "Ta bort alla?"
 #define TR_SELECT_WIDGET                "Välj widget"
 #define TR_REMOVE_WIDGET                "Ta bort widget"
 #define TR_WIDGET_SETTINGS              "Widgetinställningar"
 #define TR_REMOVE_SCREEN                "Ta bort huvudvy"
-#define TR_SETUP_WIDGETS                "Konfigurera widgets"
+
+#if LCD_W > LCD_H
+  #define TR_SETUP_WIDGETS              "Konfigurera widgets"
+#else
+  #define TR_SETUP_WIDGETS              "Konfig. widgets"
+#endif
```

Comment:
If you choose TR(short,long), this once also needs updating for consistency

Reply:
I'll leave it as is, see above reason.